keepassxc-browser/global.js

```diff
@@ -4,9 +4,9 @@ const IGNORE_NOTHING = 'ignoreNothing';
 const IGNORE_NORMAL = 'ignoreNormal';
 const IGNORE_FULL = 'ignoreFull';
 
-var schemeSegment = '(\\*|http|https|ws|wss|file|ftp)';
-var hostSegment = '(\\*|(?:\\*\\.)?(?:[^/*]+))?';
-var pathSegment = '(.*)';
+const schemeSegment = '(\\*|http|https|ws|wss|file|ftp)';
+const hostSegment = '(\\*|(?:\\*\\.)?(?:[^/*]+))?';
+const pathSegment = '(.*)';
 
 var isFirefox = function() {
     if (!(/Chrome/.test(navigator.userAgent) && /Google/.test(navigator.vendor))) {
```

Comment:
Wait `isFirefox` returns true, when it is Chrome???  :astonished:

Reply:
But if you look the parenthesis, this works just like it should. It returns true if it's not Chrome. The logic is upside down.